### PR TITLE
Removed some staff user merge code duplications

### LINKED ISSUE #1299: Code duplication in evap/staff/templates/staff_user_merge.html
In #1294 we came across some code duplication in evap/staff/templates/staff_user_merge.html. We may want to refactor that.

### PR DESCRIPTION
**Pull request by foodplanet**
Fixes #1299

### CODE REVIEW

File: evap/staff/templates/staff_user_merge.html [hunk 1]
```diff
@@ -143,209 +143,85 @@ <h3>{% trans 'Merge users' %}</h3>
...
-                                </ul>
-                            {% endfor %}
+                            {% include "staff_user_merge_course_list.html" with content_list=participation_list %}
                         </td>
                     </tr>
+
                     <tr{% if 'evaluations_voted_for' in errors %} class="table-danger"{% endif %}>
                         <td><strong>{% trans 'Voted for' %}</strong></td>
-                        {% regroup main_user.get_sorted_evaluations_voted_for by course.semester as voting_list %}
+                          {% regroup main_user.get_sorted_evaluations_voted_for by course.semester as voting_list %}
```
Comment (janno42) [inline evap/staff/templates/staff_user_merge.html]: looks like some unwanted indentation :)

File: evap/staff/templates/staff_user_merge_course_list.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% for group in content_list %}
+    {{ group.grouper }}
+    <ul>
+        {% for subgroup in group.list %}
+          {% if subgroup.full_name %}
```
Comment (janno42) [inline evap/staff/templates/staff_user_merge_course_list.html]: please use 4 spaces per indentation level

File: evap/staff/templates/staff_user_merge_course_list.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{% for group in content_list %}
```
Comment (He3lixxx) [inline evap/staff/templates/staff_user_merge_course_list.html]: As this file is not only used for courses, but also for contributions and participations and maybe for any other grouped list in the future, I suggest renaming it to `staff_user_merge_grouped_list.html`, and replacing `content_list` with `grouped_list`. What do you think?

**Comment by janno42** [comment]
Congrats on the merge of your first pull request! 🎉

**Comment by He3lixxx** [comment]
:+1:

**Pull request merged**